apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -276,7 +278,7 @@ export default function SectionsSetUpContainer({
     // TODO: this will probably eventually be a setting on the course similar to hasTextToSpeech
     // currently we're working towards piloting in Javalab in CSA only.
     const aiTutorAvailable =
-      experiments.isEnabled('ai-tutor-toggle') &&
+      experiments.isEnabled(experiments.AI_TUTOR_ACCESS) &&
```

Comment:
- [ ] instead of checking that the front-end experiment is set; pass a prop from the backend to check if the teacher is enrolled in the experiment or not so that they can't toggle the experiment on/off via enableExperiments url param